app/models/manageiq/providers/redhat/infra_manager/provision/configuration.rb

```diff
@@ -14,7 +14,7 @@ def configure_cloud_init
     return unless content = customization_template_content
     with_provider_destination { |d| d.update_cloud_init!(content) }
 
-    if Gem::Version.new(source.ext_management_system.api_version) >= Gem::Version.new("3.5.5.0")
+    if source.ext_management_system.api_version && Gem::Version.new(source.ext_management_system.api_version) >= Gem::Version.new("3.5.5.0")
```

Comment:
might be worth assigning ` source.ext_management_system.api_version` to a variable for readability, up to you though.

Reply:
:+1: good point I'll push a followup to address this